README.md

```diff
@@ -64,6 +64,30 @@ I made a [blog post][pub-setup] on how to run a server pub to relay messages to
 
 `npm run build` for developing and `npm run release` for a much smaller bundle.  You can also run `npm run inline` to genereate a single monolithic index.html file with all resources included.
 
+# Enabling WebSockets in ssb-room
+
+To run an ssb-room which this can connect to, you will need to enable WebSockets support.  This requires three things:
+
+1. Add a line to `connections` so configure the WebSockets port (external, key, and cert need to be customized for your installation):
```

Comment:
so configure -> to configure. And this probably needs to a bit more information about where this connections can be found. Might be good to explain that the key and cert are only needed for secure web sockets.

Reply:
How's that look?